tests/tck/features/match/SingleShorestPath.feature

```diff
@@ -66,34 +66,18 @@ Feature: single shortestPath
       | <("Blake Griffin" :player{age: 30, name: "Blake Griffin"})-[:like@0 {likeness: -1}]->("Chris Paul" :player{age: 33, name: "Chris Paul"})-[:like@0 {likeness: 90}]->("LeBron James" :player{age: 34, name: "LeBron James"})> |
     When executing query:
       """
-      MATCH p = shortestPath( (a:player{age:30})-[e*..5]->(b:team) ) RETURN p
+      MATCH p = shortestPath( (a:player{age:30})-[e*..5]->(b:team) ) WHERE id(a) != 'Blake Griffin' RETURN p
```

Comment:
shortest path will randomly select a result to return, the result is uncertain, so modify the result